Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -556,6 +556,61 @@ if (Intent.ACTION_VIEW.equals(intent.getAction())) {
 
 Defining and using your own URL scheme can be risky in this situation if data is sent to the scheme from an external party and processed in the app. Therefore keep in mind that data should be validated as described in "Testing custom URL schemes."
 
+### Testing for insecure Configuration of Instant Apps (MSTG-*)
+
+#### Overview
+
+With [Google Play Instant](https://developer.android.com/topic/google-play-instant/overview) you can now create Instant apps. An instant apps can be instantly launched from a browser or the "try now" button from the app store from Android API level 23 onward. They do not require any form of installation. There are a few challenges with instant apps:
+
+- there is a limited amount of size you can have with an instant app (max 10 mb).
+- there is a reduced set of permissions it is allowed to use, which is documented at the [Android Instant app documentation](https://developer.android.com/topic/google-play-instant/getting-started/instant-enabled-app-bundle?tenant=irina#request-supported-permissions).
+
+The combination of these can lead to insecure decisions, such as: stripping too much of the authorization/authentication/confidentiality logic from an app, which allows for information leakage.
+
+Note: Instant apps require an app-bundle. App-bundles are described in the [App Bundle](#App Bundle) section.
+
+#### Static Analysis
+
+Static analysis can be either done after reverse engineering a downloaded instant app, or by analyzing the app bundle. When you analyze the app bundle: check the Android Manifest to see whether `dist:module dist:instant="true"` is set for a given module (either the base or a specific module with `dist:module` set). Next, check for the various entrypoints, which entrypoints are set (by means of `<data android:path="</PATH/HERE>" />`).
+
+Now follow the entrypoints, like you would do for any Activity and check:
+
+- is there any data retrieved by the app which should require privacy protection of that data? If so, are all required controls in place?
```

Comment:
Capitals

Reply:
("Use appropriate capitalization and end punctuation for list items that individually form complete sentences.")